code/stable/dblpend/SRS/mdBook/src/SecSysContext.md

```diff
@@ -0,0 +1,21 @@
+# System Context {#Sec:SysContext}
+
+[Fig:sysCtxDiag](./SecSysContext.md#Figure:sysCtxDiag) shows the system context. A circle represents an entity external to the software, the user in this case. A rectangle represents the software system itself (DblPend). Arrows are used to show the data flow between the system and its environment.
+
+<div id="Figure:sysCtxDiag"></div>
+
+![System Context](../../../../datafiles/dblpend/SystemContextFigure.png)
```

Comment:
Did images end up working if they weren't within the `src` folder of the mdBook project?

Reply:
Nope, after researching online it seems the only way to render images is by having a copy in the `src` folder. After solving that issue, the generated links will also have to be updated to the new paths.